src/js/pdf-viewer.js

```diff
@@ -6,7 +6,7 @@ angular.module('pdf')
   function($window, $log, pdfDelegate) {
     return {
       restrict: 'E',
-      template: '<pdf-viewer-toolbar delegate-handle="{{id}}" page-count="pageCount"></pdf-viewer-toolbar><canvas></canvas>',
+      template: '<div ng-if="showToolbar"><pdf-viewer-toolbar delegate-handle="{{id}}" page-count="pageCount"></pdf-viewer-toolbar></div><canvas></canvas>',
```

Comment:
@imobach Thanks for the PR. Could you add the `ng-if` to the `pdf-viewer-toolbar` element itself. It doesn't require an additional div wrapper.